assets/_routes.json

```diff
@@ -0,0 +1,5 @@
+{
+	"version": 1,
+	"include": ["/api/*"],
```

Comment:
This file isn't technically necessary, but does allow you to specify which routes are "free"/"static" and which ones will cost ya. See: https://developers.cloudflare.com/pages/functions/routing/#functions-invocation-routes